proseco/catalog.py

```diff
@@ -76,6 +76,7 @@ def get_aca_catalog(obsid=0, **kwargs):
     :param n_guide: desired number of guide stars + monitor windows (req'd unless obsid spec'd)
     :param monitors: N x 5 float array specifying monitor windows
     :param man_angle: maneuver angle (deg)
+    :param man_angle_away: maneuver angle to attitude after this observation (deg)
```

Comment:
I think I like `man_angle_next` better, with `maneuver angle to next attitude after this observation (deg, default=180)`.

Reply:
Sure.  I don't have a preference and was just working from the slack comments.